examples/ipv6/rpl-udp/udp-client.c

```diff
@@ -40,9 +40,7 @@
 #include <stdio.h>
 #include <string.h>
 
-/* Only for TMOTE Sky? */
 #include "dev/serial-line.h"
-#include "dev/uart1.h"
```

Comment:
I though about that solution as it is done in this was in `rpl-collect` example. But I though that the examples in this directory should be independent of the platforms. And as the initialization of the UART is platform-specific, I think the correct place to do so is in the platform directory.
In fact, the include of `dev/uart?.h` is done in the corresponding `contiki-*-main.c`files. So I think that this part is unnecessary here. 
Or am I missing an important point? Just tried the given example with two TelosB nodes (real hardware): works perfectly!